album-recommendation-random-data/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do we need log4j - can't we just use Java Logging API? less dependencies is better